src/html/header-basic-mega.html

```diff
@@ -0,0 +1,41 @@
+
+  <a class="usa-skipnav" href="#main-content">Skip to main content</a>
+  <header class="usa-header usa-header-basic-megamenu" role="banner">
+    <div class="usa-disclaimer">
+      <div class="usa-disclaimer-inner">
+        <div class="usa-disclaimer-official">
+          <img src="{{ site.baseurl }}/assets/img/favicons/favicon-40.png" alt="U.S. flag">
+          <p>An official website of the United States government</p>
+        </div>
+      </div>
+    </div>
+    <div class="usa-nav-container">
+      <div class="usa-navbar">
```

Comment:
What's the difference between `navbar` and `nav`? Is there a better name for navbar, I don't really understand it in this context.